<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="EdtName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtVorname">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtPersonalnummer">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdtGehalt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="DatGeburtstag">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LblName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>85</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LblVorname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>85</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Vorname:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LblPersonalnummer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>85</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Personalnummer:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LblGehalt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>85</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Gehalt:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LblGeburtstag">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>85</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Geburtstag:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="LstAnzeige">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>380</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdAlleSehen">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Alle sehen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdEinfuegen">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Einfügen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdAendern">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ändern</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdLoeschen">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Löschen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdFiltern">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtern</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
